README.md

```diff
@@ -26,6 +45,10 @@ WebAuthn (Web Authentication) is a W3C standard for secure public-key authentica
 - WebAuthn [article with talk](https://developers.google.com/web/updates/2018/05/webauthn) in Google Developers
```

Comment:
This is pretty old by now, maybe replace with something more up to date like https://webauthn.guide/ ?